src/styles/fonts/fonts.css

```diff
@@ -9,6 +9,7 @@
   --subheading-size: 20px;
   --subheading-color: var(--gold-color);
   --subheading-spacing: 4px;
+  --subheading-weight: 300;
   --mainButton-size: 25px;
   --mainButton-color: var(--background-color);
   --mainButton-spacing: 2px;
```

Comment:
maybe put all the colors you use in one folder and the font-styling (size, spacing) in this folder

Reply:
True, Got one color.css maybe i should just define it there.